deployment/proxies.rst

```diff
@@ -78,3 +78,17 @@ other information.
 
 .. _`security groups`: http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-groups.html
 .. _`RFC 7239`: http://tools.ietf.org/html/rfc7239
+
+But what if I have Custom Headers!
+-----------------------------------
+
+Some reverse proxies (like CloudFront with ``CloudFront-Forwarded-Proto``) may force you to use a custom header.
+For instance you have ``Custom-Forwarded-Proto`` instead of ``X-Forwarded-Proto``.
+
+In this case, you'll need to set the header ``X-Forwarded-Proto`` with the value of
+``Custom-Forwarded-Proto`` early enough in your application::
+
+    // web/app.php
```

Comment:
This line should be changed to `// public/index.php` for 4.x versions.

Reply:
I made the change while merging 👍